filebeat/module/icinga/startup/test/test.log-expected.json

```diff
@@ -1,6 +1,6 @@
 [
     {
-        "@timestamp": "2018-12-11T08:08:07.894Z", 
+        "@timestamp": "2018-12-12T11:22:05.182Z", 
```

Comment:
Don't think it's a problem, but why are these timestamps still changing?

Reply:
I fixed that with https://github.com/elastic/beats/pull/9506. It's not an issue for CI.